client/src/index.jsx

```diff
@@ -117,18 +149,39 @@ class App extends React.Component {
     let targetValue = e.target.value;
     const currentValues = Object.assign({}, this.state.currentValues);
 
+    if(targetValue === ''){
```

Comment:
space between parens
```suggestion
    if (targetValue === '') {
```